demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -243,27 +240,29 @@ def _remove_builtin_scripts(self, scripts_set):
 
         return not_builtin_scripts_set
 
-    def _remove_integration_commands_scripts(self, scripts_set):
+    def _validate_checked_integration_commands_scripts(self, scripts_set):
         """
-        For each script ID in the given scripts set checks if it is an integration command by checking if it contains
-        '|||'.  If a script is not an integration command add it to a new scripts set.
-        TODO: Ignoring integration commands is temporary. Validate command should verify that each integration command
-         called from a layout, a layoutscontainer or an incident field is really exist.
-         will be fixed in: https://github.com/demisto/etc/issues/41246
-
+        For each script ID in the given scripts set checks if it is an integration command by
+        checking if it contains '|||'.  If a script is  an integration command checks in id_set.json
+        if it exists.
```

Comment:
not sure I understand the name and the explanation. is this going thru scripts or playbooks?

Reply:
I changed it, hope it is more clear
It checks whether in the scripts_set are integration commands and if there are if they are legal